src/index.ts

```diff
@@ -101,19 +101,28 @@ export class EppoJSClient extends EppoClient {
   // Ensure that the client is instantiated during class loading.
   // Use an empty memory-only configuration store until the `init` method is called,
   // to avoid serving stale data to the user.
+  /**
+   * @deprecated. use `getInstance()` instead.
+   */
```

Comment:
Guessing the plan in the future is to make this private? Or have getInstance() construct just-in time if one not set?